cli/agent_test.go

```diff
@@ -69,15 +69,15 @@ func TestWorkspaceAgent(t *testing.T) {
 			OrganizationID: user.OrganizationID,
 			OwnerID:        user.UserID,
 		})
-		dbfake.WorkspaceBuild(t, db, ws, database.WorkspaceBuild{}, &proto.Resource{
+		dbfake.NewWorkspaceBuildBuilder(t, db, ws).Resource(&proto.Resource{
 			Name: "somename",
 			Type: "someinstance",
 			Agents: []*proto.Agent{{
 				Auth: &proto.Agent_InstanceId{
 					InstanceId: instanceID,
 				},
 			}},
-		})
+		}).Do()
```

Comment:
Minor suggestion, perhaps we could rename this `s/Do/Build`, since a Builder Build's. I mean, the BuildBuilder is a bit unfortunate 😅, so I know, right, suggesting to add one more. But typically this naming goes `Do`er `Do()`, `Build`er `Build()`, etc.

Reply:
Yeah, I wanted to avoid `Build()` as it might be confusing with WorkspaceBuild.

Eventually I want the whole package to use the builder pattern and we can drop `Builder` from the function names.  So you be like

```
r := dbfake.Workspace(t, db).Do()
```